addon/components/app-header.js

```diff
@@ -1,13 +1,14 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
-import HeaderStyleMixin from '@upfluence/oss-components/mixins/header-style';
 
-export default Component.extend(HeaderStyleMixin, {
+export default Component.extend({
   tagName: 'nav',
   classNameBindings: ['headerStyleClass'],
   logo: null,
   //TODO: check if used somewhere
   indexRoute: 'index',
+  isSidebarStyle: computed.equal('headerStyle', 'sidebar'),
+  isTopbarStyle: computed.not('isSidebarStyle'),
```

Comment:
this is the content that was in the HeaderStyleMixin (which is not used elsewhere). this component itself is used in two places:
- plugin (already killed but not merged yet)
- identity-web in the connect_as (needs to be killed)